pyproject.toml

```diff
@@ -10,5 +10,14 @@ authors = [
     {name = "Baptiste Mispelon", email = "hello@bmispelon.rocks"}
 ]
 license = {file = "LICENSE"}
-classifiers = ["License :: OSI Approved :: MIT License"]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
```

Comment:
I think this should be `Development Status :: 6 - Mature`

Reply:
Excellent point